_assets/stylesheets/pages/_locations.scss

```diff
@@ -4,7 +4,7 @@
 
   @media (min-width: $screen-sm) {
     padding-bottom: 0;
-    padding-top: 135px;
+    padding-top: 32px;
```

Comment:
Why is this reduced `100px`? We should not need to make any big changes to the jumbotron. When this block is not showing we will no longer have the correct spacing for the jumbotron